### PR TITLE
ui: Add user guide link for table sampling feature

### PR DESCRIPTION
**Pull request by zhangvi7**
### Context 
Changes to make more clear to users what happens if they enable table sampling:
- Add wiki page on Querybook for table sampling user guide
     - Wiki url set in querybook/config/querybook_public_config.yaml 
- Show before and after when sample queries are applied on sampling more information click

### Test Plan
image
image

### CODE REVIEW

File: querybook/webapp/components/DataDocTableSamplingInfo/DataDocTableSamplingInfo.tsx [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+        return <Loading />;
+    } else if (error) {
+        return (
+            <ErrorMessage title={'Failed to sample query'}>
+                {formatError(error)}
+            </ErrorMessage>
+        );
+    } else {
+        return (
+            <div className="">
```
Comment (czgu) [inline querybook/webapp/components/DataDocTableSamplingInfo/DataDocTableSamplingInfo.tsx:68]: add a placeholder class name, like DataDocTableSamplingInfo

File: querybook/webapp/components/DataDocQueryCell/DataDocQueryCell.tsx [hunk 2]
```diff
@@ -946,6 +954,17 @@ class DataDocQueryCellComponent extends React.PureComponent<IProps, IState> {
         );
     }
 
+    public renderTableSamplingInfoDOM() {
+        return (
+            <DataDocTableSamplingInfo
```
[the remark this answers is not in the document]
Comment (jczhong84) [reply querybook/webapp/components/DataDocQueryCell/DataDocQueryCell.tsx:959]: +1

File: querybook/webapp/components/DataDocTableSamplingInfo/DataDocTableSamplingInfo.tsx [hunk 3]
```diff
@@ -1,82 +1,87 @@
...
         );
     } else {
-        return (
-            <div className="">
-                By enabling table sampling, it will use a sample version of the
-                table or apply table sample to the original table. Please see
-                more about it
-                <Link to={SAMPLE_USER_GUIDE_LINK}> here</Link>
+        contentDOM = (
+            <Modal onHide={onHide}>
```
[the remark this answers is not in the document]
Comment (zhangvi7) [reply querybook/webapp/components/DataDocTableSamplingInfo/DataDocTableSamplingInfo.tsx:61]: oh thanks for catching that

**Comment by zhangvi7** [comment]
- Updated for adhoc view as well
- Fix issue with sample rate getter

File: querybook/webapp/components/DataDocQueryCell/DataDocQueryCell.tsx [hunk 4]
```diff
@@ -225,7 +223,7 @@ class DataDocQueryCellComponent extends React.PureComponent<IProps, IState> {
     }
 
     public get hasSamplingTables() {
-        return Object.keys(this.samplingTables).length > 0;
+        return Object.keys(this.state.samplingTables).length > 0;
```
Comment (jczhong84) [inline querybook/webapp/components/DataDocQueryCell/DataDocQueryCell.tsx:226]: thanks for fixing it! could you help to add the sampleRate to the trackClick in the queryComposer as well? I forgot that

https://github.com/pinterest/querybook/blame/731a16635660ce1b0ad275c40abf266528b62410/querybook/webapp/components/QueryComposer/QueryComposer.tsx#L502

**Review by jczhong84 — APPROVED** [review]

**Review by czgu — APPROVED** [review]

**Pull request merged**